core/templates/dev/head/domain/exploration/SolutionObjectFactorySpec.ts

```diff
@@ -55,7 +56,29 @@ describe('Solution object factory', () => {
       });
     });
 
+    it('should create a new solution from scratch', () => {
+      var solutionFromScratch = sof.createNew(
+        true,
+        'This is the correct answer!',
+        'This is the explanation to the answer',
+        'solution');
+
+      expect(solutionFromScratch.toBackendDict()).toEqual({
```

Comment:
It's easier to read if you put all the expected objects/dicts in a separate constant and call it expectedSolution or something like that.

Reply:
Similar comments for the below tests too!

Reply:
Done.